svelte-kit-scss/src/lib/interfaces/issue-milestone.ts

```diff
@@ -0,0 +1,11 @@
+export enum IssueMilestoneState {
```

Comment:
I think we are over-typing everything here, especially with enums like this, it just causes us to have to import enums all around more.

I wonder if it would just be better to have:
```ts
export interface IssueMilestone {
  id: number;
  number: number;
  title: string;
  state: 'open' | 'closed';
}
```

at the very least we really only ever need one type to represent either `open/closed` across all the other types.

We can merge this as is, and refactor later.

Reply:
We don't need to match what they do, but also what would be functionally different if we did not use a typescript `enum`.

That schema just describes that the returned value will always be one of `"open"` or `"closed"`, so if in our code we had `state: 'open' | 'closed'` that would be correct would it not?

Reply:
It makes sense, but I do think our types are overly verbose.

> What refactoring will be needed this case?

We would just add another string literal `state:  'open' | 'closed' | 'all'` or if we were sharing a `state` type across models, we would just change it.  I don't think this is as big a deal.

But this is still fine as is.